packages/context-editor/src/stories/ContextEditor.stories.tsx

```diff
@@ -71,3 +76,64 @@ export const Blank: Story = {
 		);
 	},
 };
+
+export const WithImage: Story = {
+	args: {
+		initialDoc: docWithImage,
+		pubTypes: initialTypes,
+		pubId: "a85b4157-4a7f-40d8-bb40-d9c17a6c7a70",
+		pubTypeId: "67704c04-4f04-46e9-b93e-e3988a992a9b",
+		getPubs,
+		onChange: () => {},
+		getPubById: () => undefined,
+		atomRenderingComponent: AtomRenderer,
+		upload,
+	},
+	// Render the prosemirror doc on the screen for testing
+	render: function Render(args) {
+		const [state, setState] = useState<EditorState | undefined>(undefined);
+
+		return (
+			<>
+				<ContextEditor {...args} onChange={setState} />
+				<pre className="text-xs" data-testid="prosemirror-state">
+					{state ? JSON.stringify(state?.doc.toJSON(), null, 2) : "{}"}
+				</pre>
+			</>
+		);
+	},
+};
+
+export const ParseDOM: Story = {
```

Comment:
for trying out `parseDOM`.... the `<figure>` elements don't parse the way I want them to yet

Reply:
that's ok! I'll take a look at `parseDOM` in the table branch because I'll need to solve it for tables, too. I'll probably end up using `getContent`, but we'll see.